unison-cli/src/Unison/LSP/FileAnalysis.hs

```diff
@@ -240,9 +240,25 @@ analyseNotes fileUri ppe src notes = do
             -- still have valid diagnostics.
             TypeError.Other e@(Context.ErrorNote {cause}) -> case cause of
               Context.PatternArityMismatch loc _typ _numArgs -> singleRange loc
-              _ -> do
-                Debug.debugM Debug.LSP "No Diagnostic configured for type error: " e
-                empty
+              Context.HandlerOfUnexpectedType loc _typ -> singleRange loc
+              Context.TypeMismatch {} -> shouldHaveBeenHandled e
```

Comment:
Most of these will have been handled by specific error types above, but I expended all the cases so if new error cases are added we'll at least get a compiler warning to add them here 😄